packages/sdk/src/adapters/multipeer/protocols/clientSync.ts

```diff
@@ -254,6 +266,28 @@ export class ClientSync extends Protocols.Protocol {
         }
     }
 
+    private activeSoundInstances(actor: Partial<SyncActor>) {
+        return Promise.all([
+            (actor.activeSoundInstances || [])
+                .map(activeSoundInstance => {
+                    // TODO This sound tweaking should ideally be done on the client, because then it can consider the
+                    // time it takes for packet to arrive. This is needed for optimal timing .
+                    const targetTime = Date.now() / 1000.0;
+                    if (activeSoundInstance.message.payload.options.paused !== false) {

```

Comment:
>activeSoundInstance.message.payload.options.paused !== false [](start = 24, length = 60)

Also here. Will evaluate to true if paused is undefined. Is it possible for paused to be undefined here?